flowey/flowey_lib_hvlite/src/_jobs/cfg_versions.rs

```diff
@@ -27,7 +27,7 @@ pub const MU_MSVM: &str = "24.0.4";
 pub const NEXTEST: &str = "0.9.74";
 pub const NODEJS: &str = "18.x";
 pub const OPENHCL_KERNEL_DEV_VERSION: &str = "6.6.63.2";
```

Comment:
Will update the dev kernel once this PR is merged:
https://github.com/microsoft/OHCL-Linux-Kernel/pull/60